src/System.Device.Gpio/System/Device/Pwm/PwmPolarity.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Pwm
+{
+    /// <summary>
+    /// The polarity of the PWM signal.
+    /// </summary>
+    public enum PwmPolarity
```

Comment:
This was not in the initial concept, but came across it while coding the feature..  It seems like it would be a nice options to have and there is support in both Linux/Windows.  I can remove if decided there is no use.

Reply:
I feel using `1 - DutyCycle` is less confusing, I didn't know you added, I aleady expressed my not liking of this above :P (sorry, hopefully no hard feeling)

Reply:
> (sorry, hopefully no hard feelings)

Not the case at all.  That's what design/development is all about 😄.  Let me try another way to explain why I think this is useful.

I see your point on looking at it as `1 - Duty Cycle`, but don't think of this related to timing or the pulse.  Polarity is all about the state the logic levels for the signal.  You can verify by doing the following:

### Active High (default; also known as normal)

Reply:
Are there specific bindings in mind that require this sort of concept? There doesn't seem to be a concept of this in most libraries, although I do see a few that have it, like WinRT Apis for example. If we can make a case that some bindings won't work without this, I think it's fine to add but in general I think I agree with @krwq in which I don't see much point in adding this.

Reply:
> Are there specific bindings in mind that require this sort of concept

This is simply a basic electronics concept for hardware configurations.  Some devices require active low or high based on design functionality or design constraints.  This works for both channels that are located on dev boards and binding components.

Here are just a few links I Googled that have the PWM polarity topic.

- https://python-periphery.readthedocs.io/en/latest/pwm.html
- https://docs.microsoft.com/en-us/uwp/api/windows.devices.pwm.pwmpulsepolarity
- https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.nrf52832.ps.v1.1%2Fpwm.html
- https://lore.kernel.org/patchwork/patch/612159/

As mentioned, it can be easily removed if decided.  If that is the path, I think we at least owe it to users when [documenting the feature](https://github.com/dotnet/iot/issues/100) not having optional polarity as it would help them determine specific types of hardware to purchase and how they configure it when using the core API.

Reply:
Looks like both in Linux and Windows this functionality is supported natively by the driver, so in that case I'm fine with keeping it.